CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# win32-certstore change log
+
+Note: this log contains only changes from win32-certstore release 0.1.0 and later
+-- it does not contain the changes from prior releases. To view change history
+prior to release 0.1.0, please visit the [source repository](https://github.com/chef/win32-certstore/commits).
+
+## Latest Release 0.1.0 (2018-04-19)
+* [win32-certstore:#20](https://github.com/chef/win32-certstore/pull/20) Added support to validates a certificate in a certificate store on the basis of time validity.
```

Comment:
Please refer https://github.com/chef/chef/blob/master/CHANGELOG.md for listing issues fixed or features added in this release. This is more of an Chef standards while adding changes list.